Pipfile

```diff
@@ -22,7 +21,7 @@ pytz = "*"
 psycopg2-binary = "==2.8.2"
 sqlalchemy = "==1.3.3"
 rstr = "==2.2.6"
-psqlgraph = "==3.0.0"
+gdcdictionary = {editable = true,git = "https://github.com/NCI-GDC/gdcdictionary",ref = "release/py3"}
```

Comment:
why not gen3dictionary 2.0.1?

Reply:
not sure? This repo was using the GDC version of gdcdictionary before I upgraded to Python 3 so I thought I'd just stay consistent but it probably should use gen3dictionary instead - it's not clear to me how that dependency is supposed to work so I'm inclined to leave it for now and we can update in the future if we want.

Reply:
this repo is published on pypi as "gen3datamodel" so i think it would make sense to use the "gen3dictionary" lib instead of the branch in the gdc repo

Reply:
the `gen3datamodel` in pypi is published from the fork https://github.com/uc-cdis/gdcdatamodel - not sure how much they deviated, what do you think? Should we just use gen3datamodel and check if tests pass?

Reply:
yep you're right, I'll change this.